### PR TITLE
Update documentation and install commands

### PR DESCRIPTION
**Pull request by tamirkamara**
The following changes are proposed:
* Update install commands and Helm notes

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -49,20 +50,21 @@ If you need to build the image, please follow the [build instructions](./build.m
         ADX_TENANT_ID=[SERVICE_PRINCIPAL_TENANT_ID]
         ```
 
-        Optional - Enable ApplicationInsights telemetry
+        Optional - enable ApplicationInsights telemetry
+
         ```sh
         APPLICATION_INSIGHTS_KEY=[INSTRUMENTATION_KEY]
         COLLECT_TELEMETRY=true
```
Comment (boazsha) [inline docs/installation.md]: Should we add `enableQueryLogging`?

Comment (tamirkamara) [reply docs/installation.md]: It's optional per the recommendation to have it as an opt-in feature

Comment (guregini) [reply docs/installation.md]: But let's add it as an optional argument with '[ ]' to the command "helm install k2bridge..." : 
[--set settings.enableQueryLogging="true"]

Comment (tamirkamara) [reply docs/installation.md]: ok, so I'll remove the "optional section" since it'd be in the command

Comment (tamirkamara) [reply docs/installation.md]: Done. Please approve.

Comment (tamirkamara) [reply docs/installation.md]: I think I miss understood. enableQueryLogging is true by default, we don't want to encourage disabling it but we do document the option in the extended table linked from the main installation readme.
What do you think?

Comment (guregini) [reply docs/installation.md]: OK, so no change is needed (given enableQueryLogging is true by default)

**Pull request merged**